internal/provider/resource_aci_route_control_profile_test.go

```diff
@@ -485,7 +484,6 @@ resource "aci_route_control_profile" "test" {
   owner_key = ""
   owner_tag = ""
   route_control_profile_type = "combinable"
-  route_map_continue = "no"
```

Comment:
Why are we removing this attribute and not use the function check?

Reply:
Introduced version_mismatch in default which takes care of this.